src/builtins.rs

```diff
@@ -73,13 +74,27 @@ fn string_equals(ctx: &mut Context, args: Args) -> Option<ObjectInstance> {
 
 /// Link with a given library at runtime
 fn ffi_link_with(ctx: &mut Context, args: Args) -> Option<ObjectInstance> {
-    let lib_path = JkString::from_instance(&args[0].execute(ctx).unwrap()).0;
+    #[cfg(feature = "ffi")]
+    {
+        let lib_path = JkString::from_instance(&args[0].execute(ctx).unwrap()).0;
 
-    if let Err(e) = ffi::link_with(ctx, PathBuf::from(&lib_path)) {
-        ctx.error(e.with_msg(format!("couldn't link with library `{}`", &lib_path)));
+        if let Err(e) = ffi::link_with(ctx, PathBuf::from(&lib_path)) {
+            ctx.error(e.with_msg(format!("couldn't link with library `{}`", &lib_path)));
+        }
+
+        None
     }
 
-    None
+    #[cfg(not(feature = "ffi"))]
+    {
+        use crate::{ErrKind, Error};
+
+        ctx.error(Error::new(ErrKind::Context).with_msg(format!(
+            "Jinko is not compiled with FFI support. `t_call_invalid_builtin` is disable"
```

Comment:
```suggestion
            "jinko is not compiled with FFI support. `link_with()` is disabled"
```

`t_invalid_link_with` is the name of a test :D

Reply:
Yo soy un :monkey: